source/plugins/go/src/extension/extension_linux.go

```diff
@@ -17,8 +19,11 @@ func getExtensionConfigResponse(jsonBytes []byte) ([]byte, error) {
 
 	fs := &FluentSocket{}
 	fs.sockAddress = "/var/run/mdsd-ci/default_fluent.socket"
+	genevaLogsIntegrationEnabled := strings.TrimSpace(strings.ToLower(os.Getenv("GENEVA_LOGS_INTEGRATION")))
 	if containerType != "" && strings.Compare(strings.ToLower(containerType), "prometheussidecar") == 0 {
 		fs.sockAddress = fmt.Sprintf("/var/run/mdsd-%s/default_fluent.socket", containerType)
+	} else if genevaLogsIntegrationEnabled != "" && strings.Compare(strings.ToLower(genevaLogsIntegrationEnabled), "true") == 0 {
+		fs.sockAddress = "/var/run/mdsd-PrometheusSidecar/default_fluent.socket"
 	}
 	responseBytes, err := FluentSocketWriter.writeAndRead(fs, data)
 	defer FluentSocketWriter.disconnect(fs)

```

Comment:
Hey @ganga1980  do you want to remove the additional log message you mentioned here?

Reply:
> Hey @ganga1980 do you want to remove the additional log message you mentioned here?

Done.